superchain/extra/addresses/mainnet/op.json

```diff
@@ -11,5 +11,6 @@
   "ProxyAdminOwner": "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A",
   "SystemConfigOwner": "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A",
   "Guardian": "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A",
-  "Challenger": "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A"
+  "Challenger": "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A",
+  "ProtocolVersionsProxy":"0x8062AbC286f5e7D9428a0Ccb9AbD71e50d93b935"
```

Comment:
The actual address for both the protocol versions proxy and the superchain config proxy should go into the sueprchain.yaml file for each superchain target since these contracts apply to every chain that belongs to the target.